mmdet/datasets/coco_panoptic.py

```diff
@@ -0,0 +1,543 @@
+import json
+import os
+from collections import defaultdict
+
+import mmcv
+import numpy as np
+from mmcv.utils import print_log
+from panopticapi.evaluation import OFFSET, VOID, PQStat
```

Comment:
May need to use try mode to prompt the user to install.

Reply:
Not necessary here.